specificatie/klanten/openapi.yaml

```diff
@@ -1364,15 +1332,15 @@ components:
           - kapitaalvennootschap_binnen_eer
           - overige_buitenlandse_rechtspersoon_vennootschap
           - kapitaalvennootschap_buiten_eer
-        bezoekadres:
-          title: Bezoekadres
-          description: De gegevens over het adres van de NIET-NATUURLIJK PERSOON
-          type: string
-          maxLength: 1000
-          minLength: 1
-          nullable: true
-        verblijfBuitenland:
-          $ref: '#/components/schemas/Buitenlandsadres'
+        # bezoekadres:
+        #   title: Bezoekadres
+        #   description: De gegevens over het adres van de NIET-NATUURLIJK PERSOON
+        #   type: string
+        #   maxLength: 1000
+        #   minLength: 1
+        #   nullable: true
+        verblijfsadres:
```

Comment:
Maak hier 'verblijfadres' van. Dat is de spelling die we bij HC BRP Personen Bevragen gebruiken en volgens mij ook de juiste spelling gezien [deze bron](https://woordenlijst.org/#/?q=verblijfadres). Ook andere bronnen lijken dat te bevestigen.